carrier.py

```diff
@@ -16,17 +18,37 @@ class CarrierConfig(ModelSingleton, ModelSQL, ModelView):
     "Carrier Configuration"
     __name__ = 'carrier.configuration'
 
-    default_validation_provider = fields.Selection(
-        'get_default_validation_providers', 'Default Validation Provider'
+    default_validation_carrier = fields.Many2One(
+        'carrier', 'Default Validation Carrier'
```

Comment:
I think this need a domain for now, restricted to the carriers which can do address validation?